README.md

```diff
@@ -6,6 +6,8 @@ This library declares a `checks()` function and a `checkers` table, which
 allow to check the parameters passed to a Lua function in a fast and
 unobtrusive  way.
 
+> Starting with 2.11.0, Tarantool embeds the `checks` module. Learn more from [Module checks](https://www.tarantool.io/doc/latest/reference/reference_lua/checks/).
```

Comment:
The documentation describes both embedded and non-embedded checks features. So I don't think we should invite only 2.11.0 users to read the documentation. It should be two separate lines with different meaning.

Reply:
Now it's described as a link to a resource where you can get knowledge about "what has been changed since Tarantool 2.11.0". Quoting section, in my opinion, only intensifies the effect. I don't see why one interested only in API description would open the documentation, even though it would be useful for one.

Reply:
That's a little bit better, but I still do not understand why you do not want non-built-in checks module users to use the documentation link.

Reply:
Sorry, I don't understand what prevents non-built-in checks module users to open this link. Strictly speaking, the documentation page describes the built-in `checks` module - not the third-party `checks` module as before 2.11. And the note says exactly this. If you want to extend the description somehow, please suggest your wording, so I can add it to the PR.

Reply:
Built-in checks is exactly the same set of Lua files as non-built-in `checks` 3.3.0. And the documentation page describes built-in checks and non-built-in checks at the same time. I was expecting something like "Read more about module API at _documentation page_."

Reply:
thanks, pushed

> Built-in checks is exactly the same set of Lua files as non-built-in checks 3.3.0. And the documentation page describes built-in checks and non-built-in checks at the same time.

Yep, I understand this. But what if the versions won't match in the future, the promise that the page describes both built-in and third-party library might be broken. So I'd use more streamlined wording.

Reply:
It is expected that latest Tarantool will use latest checks